wp-config-pantheon.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Pantheon platform settings. Everything you need should already be set.
+ * To avoid merge conflicts, no changes should be introduced to this file
```

Comment:
Don't say "to avoid merge conflicts", just say that editing is not allowed.

On the Drupal side, the comment is:
```
/**
 * @file
 * Pantheon configuration file.
 *
 * IMPORTANT NOTE:
 * Do not modify this file. This file is maintained by Pantheon.
 *
 * Site-specific modifications belong in settings.php, not this file. This file
 * may change in future releases and modifications would cause conflicts when
 * attempting to apply upstream updates.
 */
```

Reply:
That was definitely a sentence I meant to circle back to (I accidentally the `.` too). Thanks.